src/components/Missions.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import MissionCard from './MissionCard';
+import { fetchMissions } from '../redux/missions/missionSlice';
+
+function Missions() {
+  const { missions, isLoading } = useSelector((store) => store.missions);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (missions.length === 0) {
+      dispatch(fetchMissions());
+    }
+  }, [dispatch, missions]);
+
+  if (isLoading) {
+    return (
+      <div className="-100 d-flex align-items-center justify-content-center">
+        <button type="button" className="btn btn-primary">
+          <span className="spinner-border spinner-border-lg" />
+        </button>
+      </div>
+    );
+  }
+  return (
+    <div className="container pt-3">
+      <table className="table table-bordered table-striped">
+        <thead>
+          <tr>
+            <th>Mission</th>
+            <th className="col-9">Description</th>
+            <th>Status</th>
+            {/* <th />  */}
```

Comment:
Great job team 🤝

- [ ] As the previous reviewer suggested, kindly remove all commented code from your project as this is regarded as a bad practice.